addons/crm/models/crm_lead.py

```diff
@@ -233,6 +234,9 @@ class Lead(models.Model):
         ('check_probability', 'check(probability >= 0 and probability <= 100)', 'The probability of closing the deal should be between 0% and 100%!')
     ]
 
+    def _compute_lang_active_count(self):
```

Comment:
Could you order the compute according to the field definition ? Like putting it after _compute_lang_id :)